### DISCUSSION TITLE
Automatic playlist creation

### DISCUSSION THREAD

**Discussion by Kouwe-sudo** [music-assistant/support#541] [Feature requests and ideas]
Nice, the new Music Assistant in Home Assistant but if I may make another suggestion for a nice improvement: Build in a button to randomly create a playlist from your own music collection (of 100 or 200 songs). I find myself falling back on my Synology with DS Audio where it's just one click to play a random playlist from my (flac-) music collection. That's nicer than listening to the same music in the same order every time. It could be coincidence but I have the idea that DS Audio even has a simple algorithm behind it because it seems like there is a system in the order of the songs.
Is something like this possible in a future release?

**Comment by marcelveldt** [comment]
That is something that I have as goal in mind yes!
For now first all basics must be covered such as managing playlists and library items, browsing, searching. Next steps would be enhance and there are endless possibilities such as using AI/ML for creating a "radio stream" of favorite tracks.

**Comment by fgibaux** [comment]
kind of related, I would love a feature like 'clear the queue and add a random album'.

**Comment by CamWynd** [comment]
I'd love a simple 'random' button that would look at my NAS and just play one random track after another (kind of a custom radio station).

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
